entity-management-solr/src/main/java/eu/europeana/entitymanagement/solr/service/SolrService.java

```diff
@@ -73,7 +73,7 @@ public void afterPropertiesSet() {
     payloadMapper.addMixIn(Agent.class, SolrEntitySuggesterMixins.AgentSuggesterMixin.class);
     payloadMapper.addMixIn(
         Organization.class, SolrEntitySuggesterMixins.OrganizationSuggesterMixin.class);
-    payloadMapper.addMixIn(Timespan.class, SolrEntitySuggesterMixins.TimespanSuggesterMixin.class);
+    payloadMapper.addMixIn(TimeSpan.class, SolrEntitySuggesterMixins.TimespanSuggesterMixin.class);
```

Comment:
we might want to update the name of the Mixin as well

Reply:
Indeed. Good catch!